pkg/disruption/backend/sampler/manifests/namespace.yaml

```diff
@@ -1,7 +1,7 @@
 kind: Namespace
 apiVersion: v1
 metadata:
-  generateName: e2e-disruption-monitor-
```

Comment:
Is there a reason for fixed name instead of generated name? Easier to do post-run cleanup?

Reply:
Fixed name is required to prevent it from running twice in the cluster. Both test and resource-observer (in some tasks) may start in-cluster monitor

Reply:
Generated names are used to ensure that different e2e tests (even with failed cleanup) can run.  While cleanup is important, not blocking start is more important.  This change needs to be revert.